packages/runtime/src/modules/AttributeListenerModule.ts

```diff
@@ -100,6 +100,10 @@ export class AttributeListenerModule extends RuntimeModule {
             metadata: { attributeListenerId: attributeListener.id }
         };
 
+        if (attribute.content["@type"] === "RelationshipAttribute" && attributeListener.peer !== attribute.shareInfo?.peer) {
```

Comment:
Dont know what this has to do with the error logger. This is only to set the third party adress when accepting an ThirdPartyQuery.

Reply:
I mean that if a peer creates a shared attribute, then the attribute listener may attempt to share that attribute with that peer again (which will not fail graciously since it then shows up in that logger). So the check `attributeListener.peer !== attribute.shareInfo?.peer` could be moved into a separate validation that will then not attempt to share the attribute.

Reply:
Yea, but I think this should be part of another PR.